src/main/features/mediation/routes/no-mediation.ts

```diff
@@ -0,0 +1,57 @@
+import * as express from 'express'
+
+import { Paths } from 'mediation/paths'
+import { Paths as responsePaths } from 'response/paths'
+import { Paths as claimantResponsePaths } from 'claimant-response/paths'
+import { ErrorHandling } from 'main/common/errorHandling'
+import { Draft } from '@hmcts/draft-store-client'
+import { MediationDraft } from 'mediation/draft/mediationDraft'
+import { FreeMediation, FreeMediationOption } from 'forms/models/freeMediation'
+import { DraftService } from 'services/draftService'
+import { User } from 'idam/user'
+import { ClaimFeatureToggles } from 'utils/claimFeatureToggles'
+import { Form } from 'forms/form'
+import { FormValidator } from 'forms/validation/formValidator'
+import { Claim } from 'claims/models/claim'
+
+function renderView (form: Form<FreeMediation>, res: express.Response): void {
+  res.render(Paths.noMediationPage.associatedView, { form: form })
+}
+
+/* tslint:disable:no-default-export */
+export default express.Router()
+  .get(Paths.noMediationPage.uri, (req: express.Request, res: express.Response) => {
+    const draft: Draft<any> = res.locals.mediationDraft
+    renderView(new Form(new FreeMediation(draft.document.willYouTryMediation.option)), res)
+  })
+  .post(
+    Paths.noMediationPage.uri,
+    FormValidator.requestHandler(FreeMediation),
+    ErrorHandling.apply(async (req: express.Request, res: express.Response) => {
+      const form: Form<FreeMediation> = req.body
+      const { externalId } = req.params
+      if (ClaimFeatureToggles.isFeatureEnabledOnClaim(res.locals.claim, 'mediationPilot')) {
```

Comment:
this is not needed here. You should only be on this page if you are in the mediation pilot which is controlled from the previous page. If you want to stop people accessing this page if they are not in the pilot then you should use a guard to redirect them. you can find examples in the guard folder in the response feature

Reply:
done